Java/Heapsort.java

```diff
@@ -0,0 +1,73 @@
+
+class HeapSort {
+    public int[] A;
+    public int heapSize; // mostly will give the index of last element in heap
```

Comment:
This one too, not very informative and misleading a bit. The comment can be removed.